docs/guides/SimpleStartupGuide.md

```diff
@@ -0,0 +1,273 @@
+
+### Install Tools
+Install the media utility tool:
+
+   ```bash
+   sudo apt install mplayer v4l-utils
+   ```
+
+### Tool Descriptions
+The table below lists command line tools this guide uses to help with EdgeX configuration and device setup.
+
+| Tool        | Description | Note |
+| ----------- | ----------- |----------- |
+| **mplayer** |  used to view the video stream | |
+| **v4l-utils** | used to determine the video stream path of a usb camera | |
+
+>Table 1: Command Line Tools
+## Get the Source Code
+> Note: This guide uses a assumes a working directory of `~/edgex`. The commands below will need to be updated if that is not the desired working directory.
+###  Download EdgeX Compose Repository
+
+1. Create a directory for the EdgeX compose repository:
+   ```bash
+   mkdir ~/edgex
+   ```
+
+2. Change into newly created directory:
+   ```bash
+   cd ~/edgex
+   ```
+
+3. Clone the EdgeX compose repository
+   ```bash
+   git clone https://github.com/edgexfoundry/edgex-compose.git
+   ```
+
+
+## Run the Service
+
+1. Run EdgeX with the microservice:  
+  - For non secure mode
+    ```
+    make run ds-usb-camera no-secty
+    ```
+  - For secure mode 
+    ```
+    make run ds-usb-camera
+    ```
+
+## Verify Service, Device Profiles, and Device
+
+1. Check the status of the container:
+
+   ```bash 
+   docker ps -f name=device-usb-camera
+   ```
+
+   The status column will indicate if the container is running and how long it has been up.
+
+   Example Output:
+
+   ```docker
+   CONTAINER ID   IMAGE                                         COMMAND                  CREATED       STATUS          PORTS                                                                                         NAMES
+   f0a1c646f324   edgexfoundry/device-usb-camera:0.0.0-dev                        "/docker-entrypoint.…"   26 hours ago   Up 20 hours   127.0.0.1:8554->8554/tcp, 127.0.0.1:59983->59983/tcp                         edgex-device-usb-camera                                                                   edgex-device-onvif-camera
+   ```
+
+2. Check that the device service is added to EdgeX:
+
+   ```bash
+   curl -s http://localhost:59881/api/v2/deviceservice/name/device-usb-camera | jq .
+   ```
+   
+   Successful:
+   ```json
+   {
+      "apiVersion": "v2",
+      "statusCode": 200,
+      "service": {
+         "created": 1658769423192,
+         "modified": 1658872893286,
+         "id": "04470def-7b5b-4362-9958-bc5ff9f54f1e",
+         "name": "device-usb-camera",
+         "baseAddress": "http://edgex-device-usb-camera:59983",
+         "adminState": "UNLOCKED"
+      }
+   }
+   ```
+   Unsuccessful:
+   ```json
+   {
+      "apiVersion": "v2",
+      "message": "fail to query device service by name device-usb-camera",
+      "statusCode": 404
+   }
+   ```
+ 
+   1. Verify device(s) have been successfully added to core-metadata.
```

Comment:
Remove indentation here to match the above lines. Right now it thinks it is a sub-line to step 2.